### ISSUE TITLE
Accessibility issue: Document does not have a main landmark

### ISSUE THREAD

**Issue by allie500** [allie500/emoji-match-game#38]
# Description

When loading the app using the Lighthouse tab in the Chrome browser inspector, we get the following notice in the Accessibility section:

> #### Document does not have a main landmark.
> One main landmark helps screen reader users navigate a web page. [Learn more about landmarks](https://dequeuniversity.com/rules/axe/4.11/landmark-one-main).

We need to add a main landmark to address this accessibility issue.

### Acceptance criteria 

- Add a main landmark to address this accessibility issue on the app's `index.html` file.
- Tests, linting, and formatting checks should all pass.
- All functionality should continue behaving as expected.

### Developer context

[Learn more about landmarks](https://dequeuniversity.com/rules/axe/4.11/landmark-one-main).
